test/ecto/migrator_test.exs

```diff
@@ -212,8 +212,10 @@ defmodule Ecto.MigratorTest do
     end
   end
 
-  test "migrations run inside a transaction by default" do
+  test "migrations run inside a transaction if the adapter supports ddl transactions" do
     capture_log fn ->
+      Process.put(:supports_ddl_transaction?, true)
```

Comment:
I think we should also set this to true here:

https://github.com/laurocaetano/ecto/blob/check-ddl-transactions/test/ecto/migrator_test.exs#L227

Otherwise we won't be testing what it is meant to test. :)